pegasus/forms/hoc_census.rb

```diff
@@ -28,6 +28,6 @@ def self.normalize(data)
   end
 
   def self.receipt
```

Comment:
Do we even this self.receipt method?

Reply:
We need to override the method on the parent (HocSignup2017)

Reply:
Got it.